src/main/kotlin/uk/gov/dluhc/printapi/mapper/CertificateMapper.kt

```diff
@@ -21,19 +21,19 @@ abstract class CertificateMapper {
 
     @Mapping(target = "id", ignore = true)
     @Mapping(target = "vacNumber", expression = "java( idFactory.vacNumber() )")
-    @Mapping(source = "localAuthority", target = "issuingAuthority")
+    @Mapping(source = "message.gssCode", target = "gssCode")
```

Comment:
A minor suggestion @Neil-Massey .

Seems like mapping of the gssCode will be done without explicitly specifying it. Also,if there is no source inferred for a target value, seems like it is ignored as well for line 22.

So we can simplify the definition of `toCertificate` method as follows:

<pre>
    @Mapping(target = "vacNumber", expression = "java( idFactory.vacNumber() )")
    @Mapping(source = "issuer.englishContactDetails.name", target = "issuingAuthority")
    abstract fun toCertificate(message: SendApplicationToPrintMessage, issuer: IssuerDto): Certificate
</pre>